pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -551,3 +553,34 @@ func (exp *flowExporter) sendDataSet() (int, error) {
 	klog.V(4).Infof("Data set sent successfully. Bytes sent: %d", sentBytes)
 	return sentBytes, nil
 }
+
+func (exp *flowExporter) findFlowType(record flowexporter.FlowRecord) uint8 {
+	if record.Conn.Mark == openflow.ServiceCTMark || exp.ipInPodSubnets(record.Conn.TupleOrig.SourceAddress) && exp.ipInPodSubnets(record.Conn.TupleOrig.DestinationAddress) {
```

Comment:
I think this has to be modified.
Do you mean the following to represent Pod-To-Pod flows and Pod-To-Service flows?
`exp.ipInPodSubnets(record.Conn.TupleOrig.SourceAddress) && (record.Conn.Mark == openflow.ServiceCTMark || exp.ipInPodSubnets(record.Conn.TupleOrig.DestinationAddress))`